kolibri/plugins/setup_wizard/api.py

```diff
@@ -297,30 +151,15 @@ def listfacilitylearners(self, request):
         :param password: Password of the user that's going to authenticate
         :return: List of the learners of the facility.
         """
-        facility_info = get_remote_users_info(request)
+        facility_id = request.data.get("facility_id")
+        baseurl = request.data.get("baseurl")
+        password = request.data.get("password")
+        username = request.data.get("username")
+        facility_info = get_remote_users_info(baseurl, facility_id, username, password)
         user_info = facility_info["user"]
         roles = user_info["roles"]
         admin_roles = (user_kinds.ADMIN, user_kinds.SUPERUSER)
         if not any(role in roles for role in admin_roles):
             raise PermissionDenied()
         students = [u for u in facility_info["users"] if not u["roles"]]
         return Response({"students": students, "admin": facility_info["user"]})
-
-
-class SetupWizardRestartZeroconf(ViewSet):
```

Comment:
Are we just using the regular device restart now, or are we triggering this update in another way?